src/node_messaging.cc

```diff
@@ -452,18 +503,29 @@ Maybe<bool> Message::Serialize(Environment* env,
     if (entry->IsObject()) {
       // See https://github.com/nodejs/node/pull/30339#issuecomment-552225353
       // for details.
-      bool untransferable;
-      if (!entry.As<Object>()->HasPrivate(
-              context,
-              env->untransferable_object_private_symbol())
-              .To(&untransferable)) {
+      bool has_transfer_mode;
+      if (!ObjectHasTransferMode(env, context, entry.As<Object>())
+               .To(&has_transfer_mode)) {
         return Nothing<bool>();
       }
-      if (untransferable) continue;
+      if (has_transfer_mode) {
```

Comment:
I am concerned about how much branch does this pull request brings to this pull request. Is performance an important factor for this for loop?